package com.xxmrk888ytxx.privatenote.Utils.Exception

class CallBackAlreadyRegisteredException() : Exception()